documentation/modules/ROOT/pages/_attributes.adoc

```diff
@@ -4,6 +4,7 @@
 :imagesdir: {moduledir}/assets/images
 :partialsdir: {moduledir}/pages/_partials
 
+:ProductName: Debezium
```

Comment:
Hi @PaulRMellor, in the MySQL docs PR, an attribute was already added called `prodname`.  Would it be possible to rebase this PR against the latest master and then decide which of the two attribute flavors we'd want to use?